src/Sfa.Das.ApprenticeshipInfoService.Api/Web.config

```diff
@@ -23,7 +23,7 @@
     <add key="ApprenticeshipIndexAlias" value="localapprenticeshipindexalias" />
     <add key="ProviderIndexAlias" value="localproviderindexalias" />
     <add key="AssessmentOrgsIndexAlias" value="localassessmentorgsindexalias" />
-    <add key="ElasticServerUrls" value="http://192.168.99.100:9200" />
+    <add key="ElasticServerUrls" value="http://127.0.0.1:9200" />
```

Comment:
it is better to have a list of urls, just in case some members of the team have a different one